manifests/simple-httpd.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: v1
+metadata:
+  name: simple-httpd
+  labels:
+    app: simple-httpd
+kind: Pod
+spec:
+  containers:
+   - name: httpd
+     image: registry.centos.org/centos/centos:7
```

Comment:
any reason to use centos here and not fedora?

Reply:
Just that it's what the other OpenShift images should be using as their base image, so just trying to share that base layer. (Though OTOH, coreos-assembler also pulls in the f28 base layer, so... I don't mind switching if you'd prefer).

Reply:
since this is the local dev case I guess it doesn't matter too much. either way works